docs/UserGuide.md

```diff
@@ -155,6 +156,19 @@ Examples:
 * `appointment-list 2024-10-17`
 * `appointment-list 2024-10-18 1000`
 
+### Making Payment for an appointment: `payment`
+
+Marks an appointment for a client as paid or unpaid.
+
+Format: `payment NAME d/DATE_AND_TIME pay/PAYMENT_STATUS`
+
+* Format for the date and time must be in yyyy-MM-dd HHmm.
+* Payment status can be in the format [true/paid or false/unpaid].
+
+Examples:
+* `payment John Doe d/2024-10-14 1200 pay/true`
+* `payment Betsy Crowe d/2024-10-14 1300 pay/unpaid`
```

Comment:
Would it be better to stick to "paid" and "unpaid" rather than "true" and "false", so it is more consistent?